src/main/flight/pid.c

```diff
@@ -496,6 +497,13 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_
     float angleTarget = angleLimit * currentPidSetpoint * maxSetpointRateInv;
     // use acro rates for the angle target in both horizon and angle modes, converted to -1 to +1 range using maxRate
 
+#ifdef USE_WING
+    if (axis == FD_PITCH) {
+        angleTarget -= (float)pidProfile->angle_pitch_offset / 10.0f;
+        // positive angle_pitch_offset will tilt the craft up - against the BF pitch convention, but more user-friendly
```

Comment:
Comment should go before code.

```suggestion
        // positive angle_pitch_offset will tilt the craft up - against the BF pitch convention, but more user-friendly
        angleTarget -= (float)pidProfile->angle_pitch_offset / 10.0f;
```